styles.css

```diff
@@ -0,0 +1,52 @@
+body{
+    text-align: center;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Helvetica Neue', Arial, sans-serif;
+    color: #333;
+    padding: 20px;
+    box-sizing: border-box;
+}
+#main{
+    padding: 20px;
+    background-color: rgb(243, 243, 243);
+    border-radius: 5px;
+}
+#img-div{
+    background-color: white;
+    padding: 20px;
+    margin: 0;
+    
+    
```

Comment:
Please try to remove unrequired indentations and spaces from your codebase for a cleaner code.